### ISSUE TITLE
Garuda Linux not detected in `OS detection`

### ISSUE THREAD

**Issue by Alonely0** [CISOfy/lynis#1122]
I'm using Garuda Linux, an Arch-based distro, and it is not detected by Lynis.

`/etc/os-release`:
```sh
NAME="Garuda Linux"
PRETTY_NAME="Garuda Linux"
ID=garuda
BUILD_ID=rolling
ANSI_COLOR="38;2;23;147;209"
HOME_URL="https://garudalinux.org/"
DOCUMENTATION_URL="https://wiki.garudalinux.org/"
SUPPORT_URL="https://forum.garudalinux.org/"
BUG_REPORT_URL="https://gitlab.com/groups/garuda-linux/"
LOGO=garudalinux
```

**Comment by sbathe** [comment]
https://github.com/CISOfy/lynis/pull/1123

**Comment by mboelen** [comment]
Fixed with #1123
